src/modules/datasets/dataset/DatasetAccessTable.tsx

```diff
@@ -0,0 +1,112 @@
+import { DETAIL_TABLE_LENGTH, TABLE_REFETCH_INTERVAL } from '@/config';
+import { useQuery } from '@tanstack/react-query';
+import { useEffect } from 'react';
+import { DataTable } from '@/components/DataTable';
+import { PaginatedNavigation } from '@/components/PaginatedNavigation';
+import { getIExec } from '@/externals/iexecSdkClient';
+import { usePageParam } from '@/hooks/usePageParam';
+import { ErrorAlert } from '@/modules/ErrorAlert';
+import { columns } from '@/modules/access/datasetColumns';
+import useUserStore from '@/stores/useUser.store';
+import { createPlaceholderDataFnForQueryKey } from '@/utils/createPlaceholderDataFnForQueryKey';
+
+function useDatasetAccessData({
+  datasetAddress,
+  currentPage,
+}: {
+  datasetAddress: string;
+  currentPage: number;
+}) {
+  const { chainId } = useUserStore();
+
+  const pageSize = DETAIL_TABLE_LENGTH * 2;
+
+  // API returns min 10 items, but we display only 8 per page
+  const apiBatch = Math.floor((currentPage * DETAIL_TABLE_LENGTH) / pageSize);
+  const startIndexInBatch = (currentPage * DETAIL_TABLE_LENGTH) % pageSize;
+
+  const queryKey = [chainId, 'dataset', 'access', datasetAddress, currentPage];
+  const { data, isLoading, isRefetching, isError, errorUpdateCount } = useQuery(
+    {
+      queryKey,
+      queryFn: async () => {
+        const iexec = await getIExec();
+
+        const { count, orders } = await iexec.orderbook.fetchDatasetOrderbook({
+          dataset: datasetAddress,
+          app: 'any',
+          workerpool: 'any',
```

Comment:
also show orders restricted by requester

```suggestion
          workerpool: 'any',
          requester: 'any',
```